index.rst

```diff
@@ -366,75 +214,46 @@ an alphabetic prefix (eg., ``v``).  This has the effect of changing the *eups*
 most recent git ref that has an *integer* as the first character.
 
-As consequene of this behavior is that the final git tag **must** be present
+As consequence of this behavior is that the final git tag **must** be present
 prior to the production of ``eupspkg``/*eups tag*.
 
-Final tag
-^^^^^^^^^
-
-XXX failures are now fatal...
-
-Now it's time to lay down the final git tag. For repositories that have already
-been branched with the ``14.0`` ref, that will fail, which is fine.
-
-This is mostly a repeat of the process for laying down the candidate tag but
-this time we use numeric tags so that eups will see them:
-
-.. code-block:: bash
-
-   # tag repos involved in the final candidate as the final build
-
-   github-tag-version \
-     --org lsst \
-     --allow-team 'Data Management' \
-     --allow-team 'DM Externals' \
-     --external-team 'DM Externals' \
-     --deny-team 'DM Auxilliaries' \
-     --debug \
-     --candidate 'v14_0_rc2 \
-     '14.0' 'b3176'
-
-
-Release build
-^^^^^^^^^^^^^
+Build and Publish
+^^^^^^^^^^^^^^^^^
 
-- Submit the run-rebuild job with your parameters (eg. ``14.0`` ``v14.0``)
+Run the jenkins `release/official-release <official-release>`_ job.
 
-- At this point you should not be seeing master-g type references as eups
-  versions. Everything should have a tag-derived version such as ``14.0`` if
-  they are a DM repo and their semantic tag (eg. ``pyfits 3.0``) if they are
-  external.  If you see one, you need to chase down why. The only situation
-  that should happen is if a third party but a branch is used for LSST
-  development that lacks any other type of semantic versioning (in the ``14.0``
-  release this included starlink_ast and jointcal_cholmod.
+For input **must only be** the latest ``rc`` tag.
+Example 1:
 
-- Note your final ``bNNNN`` number for the publish (either from the build log
-  or by looking at the next of the annotated ``14.0`` tag on any repo eg. afw).
+.. code-block:: text
 
-- Submit the run-publish job making sure you have selected ``package`` and not
-  ``git`` as the option.
+   SOURCE_GIT_REFS: v42.0.0.rc6
+   RELEASE_GIT_TAG: 42.0.0
+   O_LATEST: false
 
-Branching lsst
-^^^^^^^^^^^^^^^
+Branch ``newinstall.sh`` repo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 In this process we make use of the fact that git doesn't care whether a ref is
 a tag or a branch to constrain the number of branches to repositories that need
 retroactive maintenance or need to be available in more than one cadence. One
-such example is the ``lsst`` repo since it contains ``newinstall.sh`` which
+such example is the ``lsst`` repo since it contains newinstall.sh_ which
 sets the version of eups, and that may be different for an official release
 than the current bleed.
 
-The first repo that should be branched is ``lsst/lsst``:
+Branch `lsst/lsst <lsst>`_:
 
 .. code-block:: bash
 
    git clone https://github.com/lsst/lsst.git
-   git checkout -b 14.0
+   git checkout -b 42.0.0
```

Comment:
It is confusing calling the branch with the same name of the final release.

We still need to agree on the branch naming convention, as I already pointed out in a previous comment. In this case I suggest something like this:
b42.0
b42.0.x
v42.0.x

Reply:
I think I prefer `v42.0.x` out of the prefix/postfix options.

Reply:
fixed / made consistent.